lighthouse-core/gather/driver.js

```diff
@@ -506,12 +557,19 @@ class Driver {
       options: 'sampling-frequency=10000'  // 1000 is default and too slow.
     };
 
-    // Disable any domains that could interfere or add overhead to the trace
-    return this.sendCommand('Debugger.disable')
-      .then(_ => this.sendCommand('CSS.disable'))
-      .then(_ => this.sendCommand('DOM.disable'))
-      // Enable Page domain to wait for Page.loadEventFired
-      .then(_ => this.sendCommand('Page.enable'))
+    // Check any domains that could interfere with or add overhead to the trace.
+    if (this.domainEnabled('Debugger')) {
```

Comment:
If we're no longer disabling these before a trace, won't that change perf results?